examples/resources/konnect_api.tf

```diff
@@ -0,0 +1,6 @@
+resource "konnect_api" "my_api" {
```

Comment:
I'd expect `konnect_api` to show `attributes` as it's a common field _and_ it needs `jsonencode` which is something that could catch people out